tools/torch/toGraph.lua

```diff
@@ -0,0 +1,260 @@
+-- Copyright (c) 2015-2016, NVIDIA CORPORATION. All rights reserved.
+require 'torch'
+require 'pl'
+
+opt = lapp[[
+-n,--network (default '') path to network file
+-o,--output (default '') output path of model_def.json
+]]
```

Comment:
@TimZaman That should be pretty easy I think! As for the type info for the links, I think I was just using that for styling them (in nnTree.js). Although I don't think I used those properties in generating the graph itself.